apps/build-tools/bin/extract-api.cjs

```diff
@@ -0,0 +1,119 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+const { spawn, handleInterrupts } = require("@itwin/build-tools/scripts/utils/simpleSpawn");
+const argv = require("yargs").argv;
+const fs = require("node:fs");
+const path = require("node:path");
+
+if (argv.entry === undefined) {
+  console.log("No argument found");
+  return;
+}
+
+const isCI = process.env.TF_BUILD;
+const entryPointFileName = argv.entry;
+const missingTagsLevel = argv.missingTags ?? "error";
+const incompatibleTagsLevel = argv.ignoreIncompatibleTags ?? "error";
+
+const apiReportFolder = argv.apiReportFolder ?? "./api";
+const apiReportTempFolder = argv.apiReportTempFolder ?? "./api/temp";
+
+const config = {
+  $schema: "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  projectFolder: "../../",
```

Comment:
Shouldn't project folder be `./`? This script assumes that `cwd` is package root, right?

Reply:
This config is actually saved as a file in project's `<package_root>/lib/cjs` folder and the path is resolved from that.